### ISSUE TITLE
Fix css exports

### ISSUE THREAD

**Issue by iam-mishgun** [tomosterlund/qalendar#101]
Css exports missing on package.json
Please change exports from
```
  "exports": {
    ".": {
      "import": "./dist/qalendar.es.js",
      "require": "./dist/qalendar.umd.js"
    },
  },
```
to
```
  "exports": {
    ".": {
      "import": "./dist/qalendar.es.js",
      "require": "./dist/qalendar.umd.js"
    },
    "./dist/style.css": {
      "import": "./dist/style.css",
      "require": "./dist/style.css"
    }
  },
```

this will allow normal imports to be used

`@import "qalendar/dist/style.css";`

**Comment by tomosterlund** [comment]
Thanks for opening this issue.

I agree, this API is a nicer one, in comparison to using a relative path. Will add it.

**Comment by iam-mishgun** [comment]
thank you

**Comment by tomosterlund** [comment]
This is now available with [v1.14.1](https://github.com/tomosterlund/qalendar/releases/tag/v1.14.1)
